File: Example/InngageExampleObjc/NotificationUIExtension/NotificationViewController.m
//
//  NotificationViewController.m
//  NotificationUIExtension
//
//  Created by Augusto Cesar do Nascimento dos Reis on 04/02/21.
//  Copyright © 2021 Luis Teodoro. All rights reserved.
//

#import "NotificationViewController.h"
#import <UserNotifications/UserNotifications.h>
#import <UserNotificationsUI/UserNotificationsUI.h>
#import <Inngage/NotificationManager.h>

@interface NotificationViewController () <UNNotificationContentExtension>

@property IBOutlet UILabel *label;

@end

@implementation NotificationViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any required interface initialization here.
}

- (void)didReceiveNotification:(UNNotification *)notification {
    [NotificationManager prepareNotificationContentWithNotification:notification andViewController:self];
}

@end

File: Example/InngageExampleSwift/NotificationContentExtension/NotificationViewController.swift
//
//  NotificationViewController.swift
//  NotificationContentExtension
//
//  Created by Augusto Cesar do Nascimento dos Reis on 05/02/21.
//

import UIKit
import UserNotifications
import UserNotificationsUI
import Inngage

class NotificationViewController: UIViewController, UNNotificationContentExtension {

    @IBOutlet var label: UILabel?
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    func didReceive(_ notification: UNNotification) {
        NotificationManager.prepareNotificationContent(with: notification, andViewController: self)
    }

}
